packages/docs/getting-started.md

```diff
@@ -28,12 +28,12 @@ app.use(pinia)
 app.mount('#app')
 ```
 
-If you are using Vue 2, you also need to install a plugin and inject the created `pinia` at the root of the app:
+If you are using Vue 2, you also need to 'install' a plugin(`PiniaVuePlugin`) provided by pinia and inject the created `pinia` at the root of the app:
 
 ```js {1,3-4,12}
 import { createPinia, PiniaVuePlugin } from 'pinia'
 
-Vue.use(PiniaVuePlugin)
+Vue.use(PiniaVuePlugin) // installs the plugin
```

Comment:
let's revert this as well

Reply:
including changes in line no. 31 @posva ?